src/app/public/src/modules/image/image.component.ts

```diff
@@ -11,4 +11,7 @@ export class StacheImageComponent {
 
   @Input()
   public imageAlt: string = '';
+
+  @Input()
+  public description: string = '';
```

Comment:
Not sure description is the right label we should go with, that sounds more like something that would be added to an `alt` or `title` attribute on an image.  Perhaps something like `caption`?

Reply:
I would also leave this value as `undefined` if not specifically defined by a user. Since this is a conditional field, having a default value means this is defined.

Reply:
I 100% agree with `caption`. When looking at this I knew there was a better word for it but just couldn't think of it haha.